src/shared/validation.ts

```diff
@@ -12,8 +12,17 @@ export const NM_TOKEN_ARRAY_REGEX = new RegExp(
 );
 
 export const NM_TOKEN_DISALLOWED_CHARS_REGEX = /[^a-zA-Z0-9.\-_:]/g;
+
+export const SIRI_VM_POPULATED_STRING_REGEX = /^[^,\[\]\{\}\?$%\^=@#;:]+$/g;
```

Comment:
Since the error message below mentions only allowing certain characters, it may be better to change tighten up the regex to only allow those characters. Alternatively if this is the regex from the XSD then maybe the message to change to mention disallowed characters or reference the XSD regex

Reply:
yeah i took the regex directly from the XSD so I've updated the error message